docs/reference/configuration/single-sign-on/notifications-and-alerts.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Notifications and Alerts
```

Comment:
For now should we just call this doc `Event Logging`? Or do we want to add more alert content in here now?

Reply:
Yes, the latter! I will be expanding this section as we go through the Keycloak Notifications Project.